src/main.rs

```diff
@@ -116,29 +128,12 @@ Then try to restart Plume.
     .expect("Error setting Ctrl-c handler");
 
     let mut config = Config::active().unwrap();
-    config
-        .set_address(env::var("ROCKET_ADDRESS").unwrap_or_else(|_| "localhost".to_owned()))
-        .unwrap();
-    config.set_port(
-        env::var("ROCKET_PORT")
-            .ok()
-            .map(|s| s.parse::<u16>().unwrap())
-            .unwrap_or(7878),
-    );
-    let _ = env::var("ROCKET_SECRET_KEY").map(|k| config.set_secret_key(k).unwrap());
-    let form_size = &env::var("FORM_SIZE")
-        .unwrap_or_else(|_| "32".to_owned())
-        .parse::<u64>()
-        .unwrap();
-    let activity_size = &env::var("ACTIVITY_SIZE")
-        .unwrap_or_else(|_| "1024".to_owned())
-        .parse::<u64>()
-        .unwrap();
-    config.set_limits(
-        Limits::new()
-            .limit("forms", form_size * 1024)
-            .limit("json", activity_size * 1024),
-    );
+    config.set_address(CONFIG.rocket.address.clone()).unwrap();
```

Comment:
Isn't there a way to use the `Config` struct from rocket directly in our `config` module instead of `RocketConfig`?

Reply:
`RocketConfig` could become a `Result<rocket::Config>` (Result to denote when there was an error creating the Config, or when there is no secret key). I through of it at some point, but did not remember what I wanted to do when I was about to commit